### PR TITLE
docs(wordpress-source-plugin-tutorial): blog-post.js file name should be blog-posts.js

### PR DESCRIPTION
**Pull request by kosvrouvas**
## Description

The filename is written as blog-posts.js a few lines above and following the example throws an error on terminal.

### CODE REVIEW

**Comment by kosvrouvas** [comment]
Makes sense

File: docs/tutorial/wordpress-source-plugin-tutorial.md [hunk 1]
```diff
@@ -281,10 +281,10 @@ exports.createPages = ({ graphql, actions }) => {
     result.data.allWordpressPost.edges.forEach(({ node }) => {
       createPage({
         path: node.slug,
-        component: path.resolve(`./src/templates/blog-post.js`),
+        component: path.resolve(`./src/templates/blog-posts.js`),
```
Comment (sidharthachatterjee) [inline docs/tutorial/wordpress-source-plugin-tutorial.md]:
```suggestion
        component: path.resolve(`./src/templates/blog-post.js`),
```

File: docs/tutorial/wordpress-source-plugin-tutorial.md [hunk 2]
```diff
@@ -281,10 +281,10 @@ exports.createPages = ({ graphql, actions }) => {
...
         context: {
           // This is the $slug variable
-          // passed to blog-post.js
+          // passed to blog-posts.js
```
Comment (sidharthachatterjee) [inline docs/tutorial/wordpress-source-plugin-tutorial.md]:
```suggestion
          // passed to blog-post.js
```

**Comment by sidharthachatterjee** [comment]
@kosvrouvas Can you go ahead and update these? I noticed that you resolved the suggestions (but didn't apply them)

**Comment by kosvrouvas** [comment]
Think I did it, a bit new to github here, there are at least two more references as blog-posts.js in the article, how can I request changes on the same pull request?

**Comment by sidharthachatterjee** [comment]
@kosvrouvas No worries! You should be able to checkout this branch locally and make the changes and push it up

Not sure if you can make more changes on the same branch on GitHub itself.

**Comment by pieh** [comment]
You should be able to hit "Edit" button on https://github.com/kosvrouvas/gatsby/blob/patch-1/docs/tutorial/wordpress-source-plugin-tutorial.md - make sure to keep "commit to patch-1" checked instead of creating new PR

**Comment by kosvrouvas** [comment]
Got it! Thanks!

**Pull request merged**